doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -635,7 +635,8 @@ Note that this can be particularly cumbersome when there has been significant
 conflicting changes made on upstream main, so it is not a hard requirement.
 
 An alternative approach is to use the "squash and merge" option on the GitHub
-merge button which will reduce all commits to a single commit.
+merge button which will reduce all commits to a single commit.  Make sure that
+single commit has a DCO signature.
```

Comment:
Minor, but maybe it should say "the single commit" instead of just "single commit"?

Reply:
I believe we have disabled the squash and merge button on GitHub because it erases the DCO (and we're generally not fans).  So I'd probably just remove this paragraph once someone double-checks me.